.phpcs.xml.dist

```diff
@@ -30,6 +30,13 @@
 		</properties>
 	</rule>
 
+	<!-- Enforce array short syntax. -->
+	<rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
+
+	<rule ref="Generic.Arrays">
+		<exclude name="Generic.Arrays.DisallowShortArraySyntax"/>
```

Comment:
This exclude should go into the `WordPress-Extra` block. That ruleset includes it, so it makes sense to have the exclude there too.

Aside from that, the `<rule ref="Generic.Arrays">` now includes every single sniff in the `Generic.Arrays` category. While this currently does not cause any problems (the only other sniff there - aside from the long/short syntax sniffs - is an `ArrayIndent` sniff which apparently doesn't conflict with the WPCS one), it could cause problems in the future if more sniffs would be added to that category upstream.

Reply:
Makes sense. Fixed.